sql/core/src/main/scala/org/apache/spark/sql/execution/command/CommandUtils.scala

```diff
@@ -75,6 +77,9 @@ object CommandUtils extends Logging {
         }.sum
       }
     }
+    logInfo(s"It took ${(System.nanoTime() - startTime) / 1e6} ms to calculate the total size" +
```

Comment:
Likewise this will print something like 235.200390 ms, when you probably just want 235 here.

Reply:
Thanks for pointing it out!